core/flyway/src/main/java/io/thorntail/flyway/impl/FlywayMigrator.java

```diff
@@ -0,0 +1,32 @@
+package io.thorntail.flyway.impl;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.event.Observes;
+import javax.inject.Inject;
+
+import org.flywaydb.core.Flyway;
+
+import io.thorntail.events.LifecycleEvent;
+
+@ApplicationScoped
+public class FlywayMigrator {
+
+	@Inject
+	private FlywayConfiguration flywayConfiguration;
```

Comment:
Any reason this is marked as private?

Reply:
No, not really language-wise. Just comes as a default when I write =)

Reply:
I would recommend marking it as package private instead, for testing if nothing else.